src/views/open-api-docs-view-v2/server.ts

```diff
@@ -3,41 +3,63 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-import { getNavItems } from './utils/get-nav-items'
 import { parseAndValidateOpenApiSchema } from 'lib/api-docs/parse-and-validate-open-api-schema'
+import { stripUndefinedProperties } from 'lib/strip-undefined-props'
+import isAbsoluteUrl from 'lib/is-absolute-url'
 // Utils
 import getOperationContentProps from './components/operation-content/server'
 import getLandingContentProps from './components/landing-content/server'
+import {
+	getOperationObjects,
+	OperationObject,
+} from './utils/get-operation-objects'
+import { wordBreakCamelCase } from './utils/word-break-camel-case'
+import { groupItemsByKey } from './utils/group-items-by-key'
 // Types
 import type {
 	OpenApiDocsViewV2Props,
 	SharedProps,
+	OpenApiDocsViewV2Config,
 } from 'views/open-api-docs-view-v2/types'
 
+/**
+ * Build static props for an OpenAPI docs view.
+ *
+ * There are two main views:
+ * - Landing view, for the basePath, when no operationSlug is provided
+ * - Operation view, for the specific operationSlug that's been provided
+ */
 export async function getStaticProps({
```

Comment:
Main change here is building the sidebar link props to pass to the view. We use the newly established `getOperationObjects` and `groupItemsByKey` functions to achieve this.